lib/platform/bitbucket-server/index.ts

```diff
@@ -43,6 +43,18 @@ import { smartTruncate } from '../utils/pr-body';
 import { BbbsRestPr, BbsConfig, BbsPr, BbsRestUserRef } from './types';
 import * as utils from './utils';
 
+interface RepoResponse {
+  project: { key: string };
+  parent: string;
+  links: {
+    clone: { href: string; name: string }[];
+  };
+}
+
+interface BranchResponse {
+  displayId: string;
+}
+
```

Comment:
Move this to `./types.ts`

Reply:
done, also renamed on type that was named `Bbbs` whereas all other types are named `BbsX`